Cargo.toml

```diff
@@ -1,8 +1,23 @@
 [package]
-name = "vhost"
+name = "vhost_rs"
```

Comment:
Why did you change for `vhost_rs`?

Reply:
it conflicts with firecracker/devices/src/virtio/vhost, so a temp solution to workaround the conflict.

Reply:
What is the issue exactly?